rllib/core/rl_module/marl_module.py

```diff
@@ -144,6 +156,23 @@ def remove_module(
             self._check_module_exists(module_id)
         del self._rl_modules[module_id]
 
+    def foreach_module(
+        self, func: Callable[[RLModule, ModuleID, Optional[Any]], T], **kwargs
```

Comment:
Actually, I always found the order here very confusing in the old `foreach_policy` API. Can we flip this to:
`func: Callable[[ModuleID, RLModule, Optional[Any], T]`?